core/model/src/main/java/org/eclipse/sw360/antenna/model/coordinates/Coordinate.java

```diff
@@ -0,0 +1,224 @@
+package org.eclipse.sw360.antenna.model.coordinates;
```

Comment:
Puh, this whole wrapping is awful lot of code and since it is a pure delegate with everything PackageURL offers, it feels a lot like explicit complexity for not much gain. Does not feel right.

Reply:
What features would you be willing to sacrifice for a less complex wrapping?

Reply:
Actually my point is that it is so close to the PackageURL concept, that I do not see an easy way to replace PackageURL with something else. Since this only wraps without any logic, and transfers the PackageURL model directly I think we should use PackageURL directly knowing that this dependency then spready all over the code, but it will be an easy find and replace if we implement the class on our own in Antenna and replace the import with the local implementation to get rid of the dependency.

Reply:
After discussion ok